graphs/cmumps-patient.fbp

```diff
@@ -0,0 +1,31 @@
+# cmumps-patient.fbp
+
+# Takes a patient ID and returns a string.
```

Comment:
Discussed this with David - we agreed that I will remove this component because he is going to have Mike make changes to the translation software issues I was working around with this.